code/modules/modular_computers/hardware/lan_port.dm

```diff
@@ -0,0 +1,100 @@
+/obj/item/stock_parts/computer/lan_port
+	name = "wired connection port"
+	desc = "A basic expansion port for use with wired connections."
+	power_usage = 30
+	origin_tech = "{'programming':1,'engineering':1}"
+	icon_state = "netcard_ethernet"
+	hardware_size = 3
+	material = /decl/material/solid/glass
+	var/obj/structure/network_cable/terminal/terminal
```

Comment:
Not too sure on this one. Since the network terminal really shouldn't exist without a corresponding lan port, I feel like a direct reference is more suitable, but if others disagree I can change it.